### ISSUE TITLE
TemplateError on scrumbler/index

### ISSUE THREAD

**Issue by pprkut** [256MbTeam/Redmine-Scrumbler#51]
When accessing the index page of scrumbler within a project I get the following error:

---

Rendering scrumbler/index

ActionView::TemplateError (undefined method `compact' for #Hash:0x7f91a7b6dcb8) on line #5 of vendor/plugins/redmine_scrumbler/app/views/scrumbler/index.html.erb:
2:      <%= stylesheet_link_tag    'scrumbler', :plugin => 'redmine_scrumbler' %>
3:      <%= javascript_include_tag 'Growler-packed', :plugin => 'redmine_scrumbler' %>
4:      <%= javascript_include_tag 'growler', :plugin => 'redmine_scrumbler' %>
5:      <%= scrumbler_javascript_helper %>
6:      <%= javascript_include_tag 'scrumbler-common', :plugin => 'redmine_scrumbler' %>
7:      <%= javascript_include_tag 'scrumbler-dashboard', :plugin => 'redmine_scrumbler' %>
8: <% end %>

```
vendor/plugins/redmine_scrumbler/app/helpers/scrumbler_helper.rb:29:in `scrumbler_javascript_helper'
vendor/plugins/redmine_scrumbler/app/views/scrumbler/index.html.erb:5
app/helpers/application_helper.rb:915:in `content_for'
vendor/plugins/redmine_scrumbler/app/views/scrumbler/index.html.erb:1
/var/www/html/chucknorris.m2mobi.com/public/dispatch.fcgi:24
```

---

I'm using scrumbler 1.5.0 on ChiliProject 3.0.0

**Comment by pprkut** [comment]
This seems to be caused by the combination of scrumbler with the redmine-stuff-todo-plugin. (removing the latter fixes the error). Might indicate it's not an issue in scrumbler after all. Investigating...

**Comment by xeta** [comment]
This is 'redmine-stuff-todo-plugin' bug.
But we created temporary compatibility fix. You can get it from master branch. 
And it will be released in version 1.5.1.
